docs/release-notes/release-notes-0.17.0.md

```diff
@@ -204,6 +204,9 @@ compare the default values between lnd and sample-lnd.conf.
 * [Updated bbolt to v1.3.7](https://github.com/lightningnetwork/lnd/pull/7796)
   in order to address mmap issues affecting certain older iPhone devices.
 
+* [Remove rebroadcasting of
```

Comment:
For this change we need more detailed notes mentioning the removal of the db bucket.